pyrasterframes/src/main/python/docs/time-series.pymd

```diff
@@ -35,48 +35,48 @@ park_geo = park_df[park_df.UNIT_CODE=='CUVA'].geometry[0]
-                    (cat.acquisition_date < lit('2018-07-01')) 
+                    (cat.acquisition_date < lit('2018-07-01'))
             )
 park_cat.printSchema()
 ```
 
 ## Catalog Read
 
-Now we have a catalog with several months of MODIS data for a single granule. However the granule is larger than our park boundary. We will combine the park geometry with the catalog, and read only the bands of interest to compute NDVI, which we discussed @ref:[earlier](local-algebra.md#computing-ndvi).
+Now we have a catalog with several months of MODIS data for a single granule. However, the granule is larger than our park boundary. We will combine the park geometry with the catalog, and read only the bands of interest to compute NDVI, which we discussed in a @ref:[previous section](local-algebra.md#computing-ndvi).
 
 ```python read_catalog
 raster_cols = ['B01', 'B02',] # red and near-infrared respectively
-park_rf = spark.read.raster(catalog=park_cat.select(['acquisition_date', 'granule_id'] + raster_cols), 
+park_rf = spark.read.raster(catalog=park_cat.select(['acquisition_date', 'granule_id'] + raster_cols),
                             catalog_col_names=raster_cols,
                            ) \
-                    .withColumn('park', st_geomFromWKT(lit(park_geo.wkt))) 
+                    .withColumn('park', st_geomFromWKT(lit(park_geo.wkt)))
 park_rf.createOrReplaceTempView('catalog_read')
 park_rf.printSchema()
 ```
 
 ## Vector and Raster Data Interaction
 
-Now we have the vector representation of the park boundary alongside the tiles of red and near infrared bands. Next we need to create a tile representation of the park to allow us to limit the time series analysis to pixels within the park. This is similar to the masking operation demonstrated in @ref:[NoData handling](nodata-handling.md#masking).
+Now we have the vector representation of the park boundary alongside the _tiles_ of red and near infrared bands. Next, we need to create a _tile_ representation of the park to allow us to limit the time series analysis to pixels within the park. This is similar to the masking operation demonstrated in @ref:[NoData handling](nodata-handling.md#masking).
 
-We do this using two SQL transformation. This demonstrates that we can use any `rf_` or `st_` function in SQL. The first one will reproject the park boundary from coordinates to the MODIS sinusoidal projection. The second one will create a new tile aligned with the imagery containing value 1 where the pixels are contained within the park and NoData elsewhere.
+We do this using two SQL transformations. The first one will reproject the park boundary from coordinates to the MODIS sinusoidal projection. The second one will create a new _tile_ aligned with the imagery containing a value of 1 where the pixels are contained within the park and NoData elsewhere. The code demonstrates that we can use any `rf_` or `st_` function in SQL.
```

Comment:
+1 

We could also add a few words with a link to the languages.md page here. 

Maybe something like.

> For a more thorough demonstration of how Python and SQL code compare in RasterFrames see the @ref:[language support section](languages.md).